project-management/platform-requirements/fujitsu-business-requirements.md

```diff
@@ -0,0 +1,126 @@
+
+### Agile and open source development
+
+The Fujitsu team will work following an [Agile and open source development process](project-management/agile-opensource-workflow.md).
+
+The Fujitsu team will publish a [sprint report](/project-management/sprint-demo-requirements.md) and associated demonstration video at the end of each sprint to maintain transparent engagement with the broader autistic and developer communities, and with the people who lie at the intersection of the two.
+
+The [sprint report](/project-management/sprint-demo-requirements.md) will containing a list of issues that have been [closed](/project-management/sprint-demo-requirements.md#closed-issues), [progressed](/project-management/sprint-demo-requirements.md#progressed-issues) or [opened](/project-management/sprint-demo-requirements.md#opened-issues) during the sprint, [challenges](/project-management/sprint-demo-requirements.md#challenges) that the team faced, and requests for [community feedback](/project-management/sprint-demo-requirements.md#community-feedback-requests).
+
+The [video summary](/project-management/sprint-demo-requirements.md#video) of the sprint report will give a public and accessible overview of the work that was done during the sprint.
+
+## User experience research
+
+To ensure that the website is accessible and useful, the website will be designed and built in close collaboration with autistic people and members of the broader autism community.
+
+### User journeys
+
+The platform will be developed based on the development and continual refinement of user journeys.
+
+The initial list of platform users are:
+
+* People on the autistic spectrum.
+* People with different accessibility requirements, including those who are dyslexic and non-verbal.
+* Parents and carers of autistic people.
+* People who are both autistic themselves and the carers or parents of autistic people.
+* People who can use the data to directly influence things like policy or the built environment.
+* People who will moderate uploaded content.
+* Autism researchers.
+* Natural language processing experts who will work with the data to link similar experiences together.
+* Members of a future production, deployment and maintenance team.
+
+### 1:1 testing and engagement
+
+Fujitsu will be responsible for planning, undertaking and reporting on monthly user experience research sessions.
+
+These user testing sessions should reach a wide and diverse group.
+For each session, identifying and proactively researching those who are not involved should be part of the planning and iteration of the user design, so the interface can be as inclusive as possible.
+
+We expect approximately three user research sessions will be held monthly with autistic people, their families and carers.
+All insights from the user research session should be captured in the project GitHub after review by the user.
+
```

Comment:
Autistica's Insight group is a fantastic resource for this - Bethan Davies can put the Fujitsu team in touch with people for user testing. Keep Georgia Aitkenhead in the loop as she can also connect members of Fujitsu with project volunteers who are not members of the Insight group (who may wish to be involved)